bin/process-csv.js

```diff
@@ -31,143 +31,163 @@
 
     // Generate "products.csv" based on the source data.
     processProducts = function (workPath, targetPath) {
-        var skippedRecords = [];
+        var skippedRecords = [],
+            authors = {};
         console.log('Processing CSV data with workPath="%s", targetPath="%s"', workPath, targetPath);
         fs.readdir(targetPath + '/images', function (err, imageFilenames) {
             if (err) {
                 console.error('Could not read images path "%s", please check the path exists and try again', targetPath + '/images');
                 return;
             }
-            fs.createReadStream(workPath + '/Books.csv')
+            fs.createReadStream(workPath + '/Authors.csv')
```

Comment:
First read the Authors file and convert it into a map of AuthorID to composed name.